README.md

```diff
@@ -6,6 +6,12 @@
 
 A wrapper around [TensorFlow](https://www.tensorflow.org/), a popular open source machine learning framework from Google.
 
+| ⚠️ Notice ⚠️ |
+| --- |
+| **Tensorflow.jl is in minimal maintance mode** |
+| While it works, it is not recieiving new features, and is  bound to an old version of libtensorflow. |
+| New users are strongly recommended to start-out with a more modern and idiomatic julia deep-learning framework such as [Flux.jl](https://github.com/FluxML/Flux.jl). |
```

Comment:
" New users are strongly recommended to start-out with a more modern and idiomatic julia" - this sounds a little too opinionated to me for a README (although I happen to agree with it) - what about a more neutral comment like "the authors recommend Flux for new users"?

Reply:
sure 
```suggestion
| The authors recommend [Flux.jl](https://github.com/FluxML/Flux.jl) for new users. |
```